nix_update/eval.py

```diff
@@ -142,13 +142,22 @@ def eval_expression(
     return f"""
 let
   {indent(dedent(let_bindings), "  ")}
+  positionFromMeta = pkg: let
+    parts = builtins.match "(.*):([0-9]+)" pkg.meta.position;
+  in if pkg ? meta.position then {{
```

Comment:
Do we need this `if` here? It's already checked on line 158. Also would make the code slightly simpler.